### PR TITLE
AudioPlayerAgent: Update ver 1.3

### PR DESCRIPTION
**Pull request by JeanTracker**
AudioPlayer v1.3 complements the synchronization of the content
being played on the Media App Center and the device.

Signed-off-by: JeanTracker <hyojoong.kim.jean@gmail.com>

### CODE REVIEW

File: src/capability/audio_player_agent.cc [hunk 1]
```diff
@@ -1039,6 +1055,65 @@ void AudioPlayerAgent::parsingRequestPlayCommand(const char* dname, const char*
     sendEventRequestPlayCommandIssued(dname, payload);
 }
 
+void AudioPlayerAgent::parsingRequestOthersCommand(const char* dname, const char* message)
+{
+    Json::Value root;
+    Json::Reader reader;
+    std::string play_service_id;
+    std::string payload;
```
Comment (kimhyungrok) [inline src/capability/audio_player_agent.cc:1063]: The "std::string payload" variable is not used. Please remove it.

File: src/capability/audio_player_agent.cc [hunk 2]
```diff
@@ -589,12 +589,28 @@ void AudioPlayerAgent::sendEventRequestPlayCommandIssued(const std::string& dnam
...
 }
 
-void AudioPlayerAgent::sendEventByRequestDirective(const std::string& dname, EventResultCallback cb)
+void AudioPlayerAgent::sendEventByRequestOthersDirective(const std::string& dname, EventResultCallback cb)
 {
     std::string ename = dname + "Issued";
     sendEventCommon(ename, std::move(cb));
 }
 
+void AudioPlayerAgent::sendEventRequestCommandFailed(const std::string& dname, const std::string& play_service_id, const std::string& type, const std::string& reason, EventResultCallback cb)
```
Comment (kimhyungrok) [inline src/capability/audio_player_agent.cc:598]: The "const std::string& dname" and "const std::string& play_service_id" parameters are not used in this function. Please check and remove it. Or maybe, it not correct assign ps_id to "root["playServiceId"]".  (Reconsider user play_service_id.)

Comment (JeanTracker) [reply src/capability/audio_player_agent.cc:598]: Thanks. I modified.

**Comment by JeanTracker** [comment]
This PR is closed due to pending specification decisions.

**Pull request closed without merging**